### ISSUE TITLE
Ubuntu 20.04 missing required dependency 'setfacl'

### ISSUE THREAD

**Issue by nockstarr** [splunk/ansible-role-for-splunk#106]
I have deployed Splunk multiple times on CentOS 7 using this repo without any trouble.
Attempted to deploy on a fresh Ubuntu 20.04 Server today and got the following error:

```
TASK [../roles/splunk : Set default facl to allow splunk user to read /var/log] *******************************************************************************************************************************************************
task path: /home/user/GitHub/ansible-role-for-splunk/roles/splunk/tasks/configure_facl.yml:4
...
"msg": "Failed to find required executable \"setfacl\" in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin"
...
```

Seems like Ubuntu is no longer shipping with ``acl``... Nothing worse than ``sudo apt-get install acl`` and its all good.

Maybe ``acl`` should be added to added to a "dependency" task or noted in REAME.MD? If not its searchable here:)

Other minor observations if you have opted for ``install_utilities: true`` the following packages is not found in apt:
1. policycoreutils-python
2. bind-utils
3. setroubleshoot

The Ubuntu equivalent packages are:
| CentOS 7               | CentOS 8                     | Ubuntu 20.04                                                                                                  |
|------------------------|------------------------------|---------------------------------------------------------------------------------------------------------------|
| policycoreutils-python | policycoreutils-python-utils | policycoreutils-python-utils                                                                                  |
| bind-utils             | bind-utils                   | dnsutils                                                                                                      |
| setroubleshoot         | setroubleshoot               | Seems to be part of "selinux-policy-default package" (likely something you dont want to install unknownengly) |

**Comment by drewlap** [comment]
`- name: Install a list of packages
  become: true
  ansible.builtin.apt:
    pkg:
    - acl`

My workaround added to the top of install_splunk.yml

**Comment by dtwersky** [comment]
Fixed in #138 and #149
